akka-http-core/src/main/resources/reference.conf

```diff
@@ -404,4 +404,6 @@ akka.http {
     # HttpResponse on client side).
     tls-session-info-header = off
   }
+
+  strict-entity-max-bytes = 9223372036854775807
```

Comment:
Please note that this is under ` akka.http {` https://github.com/akka/akka-http/pull/706/files#diff-b4fbbc15badf58a0b02c9fd961f0aa2fR12